docs/s3cmd-with-minio.md

```diff
@@ -1,105 +1,176 @@
+```
+
+**Note:** Accept the defaults when prompted.
+
+### 3.2 Edit the Configuration File
+Modify the configuration file to enable `S3cmd` to manage buckets on https://play.minio.io:9000:
+
+#### 3.2.1. Navigate to **/users/<your user name>** and open **.s3cfg** in a text editor.
+#### 3.2.2. Edit the following fields in **.s3cfg** to configure the endpoint:
+
+```sh
+bucket_location = us-east-1
 host_base = play.minio.io:9000
 host_bucket = play.minio.io:9000
-bucket_location = us-east-1
 use_https = True
+```
 
-# Setup access keys
+#### 3.3.3. Edit the following fields in **.s3cfg** to set the access keys:
+
+```sh
 access_key =  Q3AM3UQ867SPQQA43P2F
 secret_key = zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
+```
+
+#### 3.3.4. Edit the following fields in **.s3cfg** to enable S3 v4 signature APIs:
 
-# Enable S3 v4 signature APIs
+```sh
 signature_v2 = False
 ```
 
-## 4. Commands
+#### 3.3.5. Save **.s3cfg**.
 
-### To make a bucket
+**Note:** The variables set in this example are for public testing and development on <https://play.minio.io:9000>. Modify these variables as appropriate when developing for your own Minio Server.
+
+
+## <a name="runs3cmdcommands"></a>4. Examples of Typical `S3cmd` Commands
+Navigate to the installation directory for **S3cmd** and invoke the following commands to create and manage a bucket:
+
+### Create a Bucket
 
 ```sh
 s3cmd mb s3://mybucket
+```
+
+You should see a response similar to this one:
+
+```sh
 Bucket 's3://mybucket/' created
 ```
 
-### To copy an object to bucket
+### Copy an Object to the Bucket
 
 ```sh
 s3cmd put newfile s3://testbucket
+```
+
+You should see a response similar to this one:
+
+```sh
 upload: 'newfile' -> 's3://testbucket/newfile'  
 ```
 
-### To copy an object to local system
+### Copy an Object to the Local File System
 
 ```sh
 s3cmd get s3://testbucket/newfile
+```
+
+You should see a response similar to this one:
+
+```sh
 download: 's3://testbucket/newfile' -> './newfile'
 ```
 
-### To sync local file/directory to a bucket
+### Sync a Local File/Directory to a Bucket
 
 ```sh
 s3cmd sync newdemo s3://testbucket
+```
+
+You should see a response similar to this one:
+
+```sh
 upload: 'newdemo/newdemofile.txt' -> 's3://testbucket/newdemo/newdemofile.txt'
 ```
 
-### To sync bucket or object with local filesystem
+### Sync a Bucket with the Local File System
 
 ```sh
 s3cmd sync  s3://testbucket otherlocalbucket
+```
+
+You should see a response similar to this one:
+
+```sh
 download: 's3://testbucket/cat.jpg' -> 'otherlocalbucket/cat.jpg'
 ```
 
-### To list buckets
+### List all Buckets
 
 ```sh
 s3cmd ls s3://
+```
+
+You should see a response similar to this one:
+
+```sh
 2015-12-09 16:12  s3://testbbucket
 ```
 
-### To list contents inside bucket
+### List the Contents of a Bucket
 
 ```sh
 s3cmd ls s3://testbucket/
+```
+
+You should see a response similar to this one:
+
+```sh
                                       DIR   s3://testbucket/test/
```

Comment:
Is this formatted? the DIR seems to be off. Can we reformat this to be in one line?

Reply:
@deekoder , @eco-minio  I have made these changes in the branch. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-2/docs/s3cmd-with-minio.md.